libs/designsystem/src/lib/components/accordion/accordion-item.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container>
+  <div
+    (click)="isExpanded = !isExpanded"
+    class="header"
+    role="button"
+    [class.expanded]="isExpanded"
+    [attr.aria-expanded]="isExpanded"
+    [attr.aria-controls]="_contentId"
+    [id]="_titleId"
+  >
+    <div class="title">{{ title }}</div>
+    <kirby-icon name="arrow-down"></kirby-icon>
+  </div>
+  <div
+    class="content"
+    role="region"
+    [attr.aria-labelledby]="_titleId"
+    [id]="_contentId"
+    [@isExpanded]="!!isExpanded"
+  >
+    <div class="content-body">
```

Comment:
Is content-body container necessary?

Reply:
Yes. We do need it to give the hidden content some padding. I have tried to not have it.
- I cannot give the parent (class="content") the padding instead, because then the padding is given even if the content is not shown.
- I cannot give the parent the padding on expanded, because you will then see the content without padding while animating closing the accordion.